samples/word/50-document/insert-external-document.yaml

```
@@ -0,0 +1,101 @@
+order: 2
+id: word-document-insert-external-document
+name: Insert an external document
+description: Inserts an external document into the current document.
```

Clarify this is text-only, not formatting.
```suggestion
description: Inserts the text of an external document into the current document.
```